src/property-filter/interfaces.ts

```diff
@@ -20,6 +20,7 @@ import {
   PropertyFilterProperty,
   PropertyFilterToken,
 } from '@cloudscape-design/collection-hooks';
+import { PropertyFilterTokenGroup } from '@cloudscape-design/collection-hooks/cjs/interfaces';
```

Comment:
These are temporary changes just to prove that the PR indeed fetches code from the collection-hooks branch: https://github.com/cloudscape-design/collection-hooks/pull/74